src/core/src/tensor_util.cpp

```diff
@@ -28,3 +33,95 @@ bool ov::util::reduce_and(const ov::Tensor& t) {
         return false;
     return outputs[0].data<char>();
 }
+
+namespace ov::util {
+namespace {
```

Comment:
This anonymous names probably can be removed.

Reply:
anonymous namespace maybe usefull for unity build. Not in this case but in general